documentation/writing_yaml_plans.md

```diff
@@ -124,6 +124,34 @@ prints the object as a string.
 For information on printing a step result with `message`, see [Debugging
 plans](#debugging-plans).
 
+
+### Verbose step
```

Comment:
@hestonhoffman  Currently this is just a port of `message` step right above it. I figured a lot of it would be able to be "similar to `message` step...." but I wasn't sure how much. Any advice?

Reply:
hi @artlawson, it's better not to use "similar to..." language. It's not a great user experience to find the thing you're looking for and then have to go somewhere else. It's also problematic over time because the parity between things can shift and create a maintenance burden. Copying another section and changing the relevant information is a great approach for something like this. It keeps the docs consistent, and it gives the user the info they're looking for 👍

Reply:
Having said that, if the two sections are very similar and are linked in functionality, we might have an opportunity to combine the two sections 🤔 I'll take another look when I review.

Reply:
Ok great! Thanks for the insight :)